### DISCUSSION TITLE
Using Command Line To Reduce Triangle Count in File

### DISCUSSION THREAD

**Discussion by annechanchan** [zeux/meshoptimizer#511] [Q&A] [answered]
Hi, I am trying to compress my GLB file to upload into Spatial. In this [Spatial article](https://support.spatial.io/hc/en-us/articles/360049861591-Optimizing-3D-Models), they recommend we run the below command line to reduce the triangle count.
`gltfpack -si 0.2 -i input.glb -o output.glb`, 

I have globally installed both GLTFPACK and DRACP 3D Graphic Data Compression, but am getting the below error message. Was wondering if there is anything I am missing in the command or if there are any steps I am missing to integrate Draco mesh compression to gltfpack. 

<img width="807" alt="Screen Shot 2022-11-14 at 11 06 31 AM" src="https://user-images.githubusercontent.com/89041409/201746255-cc3d1d12-f255-4bb8-be09-85dad484fc36.png">

**Comment by zeux** [comment, marked as answer]
gltfpack can not read files that use Draco encoded meshes (and won't be able to in the future). I'd suggest getting the source .glb file that doesn't use Draco or, if that is unavailable, I think glTF-Transform might be able to decompress Draco compressed files.

**Reply by annechanchan** [reply]
Thank you
